src/lib/receiver.js

```diff
@@ -49,6 +56,7 @@ function createReceiver (opts) {
   const sspReceiverId = SSP_RECEIVER_ID_PREFIX + receiverId + '.'
   debug('receiver id: ' + receiverId)
 
+  const reviewPayment = opts.reviewPayment || (() => Promise.resolve(null))
```

Comment:
This is one reason having a receiver listening for SSP/KEP and IPR payments at the same time is problematic. I think the review payment function should be required to enable SSP/KEP payments, and we could do that without making a breaking change